codama-koroks/src/item_korok.rs

```diff
@@ -9,10 +10,12 @@ use std::ops::AddAssign;
 
 #[derive(Debug, PartialEq)]
 pub enum ItemKorok<'a> {
-    FileModule(FileModuleKorok<'a>),
-    Module(ModuleKorok<'a>),
-    Struct(StructKorok<'a>),
+    FileModule(Box<FileModuleKorok<'a>>),
+    Module(Box<ModuleKorok<'a>>),
+    Struct(Box<StructKorok<'a>>),
```

Comment:
Can you tell me more about why we need to box these now? These were already recursive variants before and did not require boxing.